src/interfaces/IQuoter.sol

```diff
@@ -74,26 +56,20 @@ interface IQuoter {
     /// exactAmount The desired output amount
     /// sqrtPriceLimitX96 The price limit of the pool that cannot be exceeded by the swap
     /// hookData arbitrary hookData to pass into the associated hooks
-    /// @return deltaAmounts Delta amounts resulted from the swap
-    /// @return sqrtPriceX96After The sqrt price of the pool after the swap
-    /// @return initializedTicksLoaded The number of initialized ticks that the swap loaded
+    /// @return amountIn The input quote for the exactOut swap
+    /// @return gasEstimate Estimated gas units used for the swap
     function quoteExactOutputSingle(QuoteExactSingleParams memory params)
```

Comment:
does it make sense to add batched functions so you can get many quotes in one call?

Reply:
they use an external multicall contract for multicalling quoters onchain 
on mainnet its [here](https://etherscan.io/address/0x1F98415757620B543A52E61c46B32eB19261F984#code)

Reply:
ah yeah cool